lib/bolt/node.rb

```diff
@@ -49,6 +49,22 @@ def initialize(target, config: Bolt::Config.new)
     def uri
       @target.uri
     end
+
+    def upload(_source, _destination)
+      raise 'Not implemented'
```

Comment:
Should these raise a `NotImplementedError`?

Reply:
I'll do that, I was being lazy and it was mainly for documentation of function signature.

Reply:
We're getting rid of the base class soon anyway...